src/base/rss/rss_feed.cpp

```diff
@@ -193,6 +193,8 @@ bool Feed::hasError() const
 
 void Feed::handleDownloadFinished(const Net::DownloadResult &result)
 {
+    m_downloadHandler = nullptr; // will be deleted by DownloadManager later
```

Comment:
Just to be sure, you mean the object is managed by DownloadManager right?

Reply:
Yes.